kube-core/src/conversion/types.rs

```diff
@@ -13,7 +13,7 @@ pub const META_API_VERSION_V1: &str = "apiextensions.k8s.io/v1";
 pub struct ConvertConversionReviewError;
 
 /// Struct that describes both request and response
-#[derive(Serialize, Deserialize)]
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
Eq might be slightly too strong since these objects contain json which contain floating points which are not Eq.

It's also standard k8s-openapi convention to just do PartialEq.

Reply:
That's a good point, thanks! Removed `Eq` from the PR